python-sdk/tests/benchmark/infrastructure/kubernetes/job.yaml

```diff
@@ -17,7 +17,16 @@ spec:
             memory: 6Gi
           limits:
             memory: 10Gi
+        volumeMounts:
+          # mount the nfs volume for local files
+          - name: benchmark-pvc
+            mountPath: "/mnt"
```

Comment:
@sunank200 can you specify the path to local datasets?

Reply:
Yes its changed to `/data`